.github/workflows/docs-build-push.yml

```
@@ -38,6 +38,12 @@ on:
         type: string
         description: "Overrides default of latest hugo theme. Useful for testing pre-release versions. Must start with 'v' before version."
         default: ""
+      auto_deploy_branch:
+        type: string
+        description: "Branch specified here will autodeploy to the env specified in auto_deploy_env. An on.push for this branch must be specified in caller."
```

```suggestion
        description: "A branch specified here will autodeploy to the environment specified in auto_deploy_env. An on.push event for this branch must be specified in caller."
```

We don't have a strict budget on letters. If it's description a human will probably read, avoid shorthand explanations.